1. Developer basics/5_shell_scripting/5-1_bash_shell_zsh.md

```diff
@@ -45,9 +66,231 @@ For the most part bash and zsh are almost identical which is a relief. Navigatio
 | Auto completion works faster.                                                                 | Auto completion is slower.                                                    |
 | Smaller community                                                                             | Wider community                                                               |
 
-<br />
+## How to install and set up Zsh as default
+
+Installation on many platforms and setting up Zsh as default you can
+find [here](https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH#install-and-set-up-zsh-as-default)
+.
+
+## Contents
```

Comment:
Maybe we can have the contents before the intro and how to install?

Reply:
Change done! @Wason1797